System/RTEError.cpp

```diff
@@ -24,29 +24,17 @@ namespace RTE {
 
 		std::string abortMessage;
 
-#ifndef RELEASE_BUILD
 		// Show message box with explanation
 		abortMessage = "Runtime Error in file " + file + ", line " + std::to_string(line) + ", because:\n\n" + description + "\n\nThe last frame has been dumped to 'AbortScreen.bmp'";
```

Comment:
I'm doing this because it's much better for bug reports if we get the full explanation, even if the user doesn't understand.